src/main/java/seedu/storemando/logic/parser/ReminderCommandParser.java

```diff
@@ -19,21 +36,26 @@
      */
     @Override
     public ReminderCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ReminderCommand.MESSAGE_USAGE));
-        }
+        try {
+            long numOfDayFromToday = 0;
+            String trimmedArgs = args.trim();
+            String[] stringArgsArr = trimmedArgs.split(" ");
+            if (stringArgsArr.length == 1) {
+                numOfDayFromToday = Long.parseLong(stringArgsArr[0]);
+            } else if (stringArgsArr.length == 2) {
+                Long timeValue = Long.parseLong(stringArgsArr[0]);
+                String timeUnit = stringArgsArr[1];
+                numOfDayFromToday = timeConversation(timeValue, timeUnit);
```

Comment:
for this whole parsing right, perhaps you may wanna think about changing it to a prefix kinda thing. Say reminder d/3 reminder w/2, with that you will be able to use preambles and also prefix isPresent to do most of the checks here and I can foresee the code to be neater :)

Reply:
I feel like prefixes should solely be used to detect each attribute of the items. Whereas this is more of detecting a keyword.

Reply:
I agree with Jay as well. I think it would be less ambiguous and would be standardised across the commands.

Reply:
If you all want it to be in prefixes, sure. But I will be doing it on another branch "Modify Reminder feature". To add in prefixes.

Reply:
All right because initially, the plan we discussed was to make the input more lenient to allow the user to input "reminder 3 days". But I can make it work using prefixes as well, but I probably going to do it on another branch "Modify Reminder Feature" for v1.3